tests/test_zmq.py

```diff
@@ -57,8 +57,9 @@ class TestZmqDash(object):
     @pytest.fixture(scope="class")
     def enable_orchagent_zmq(self, dvs):
         # change orchagent to use ZMQ
+        # change orchagent to use custom create_switch_timeout
         dvs.runcmd("cp /usr/bin/orchagent.sh /usr/bin/orchagent.sh_zmq_ut_backup")
-        dvs.runcmd("sed -i.bak 's/\/usr\/bin\/orchagent /\/usr\/bin\/orchagent -q tcp:\/\/127.0.0.1:8100 /g' /usr/bin/orchagent.sh")
+        dvs.runcmd("sed -i.bak 's/\/usr\/bin\/orchagent /\/usr\/bin\/orchagent -q tcp:\/\/127.0.0.1:8100 -t 60 /g' /usr/bin/orchagent.sh")
```

Comment:
why this specific test is changed?

Reply:
As the change is in main() passing a new argument to orchagent, found we that we are doing a similar argument change in this test case. This change does not affect the behavior of the test as timeout value we are sending is still same as default(60 sec). But helps to cover the UT for new code change.